optimize/upgrade/pom.xml

```diff
@@ -52,7 +54,6 @@
     <dependency>
       <groupId>org.apache.commons</groupId>
       <artifactId>commons-lang3</artifactId>
-      <version>3.14.0</version>
```

Comment:
❌ I think you should not remove the version declaration, it is not defined in dependency management.